tests/README.md

```diff
@@ -47,6 +56,76 @@ ssh on the libvirt domain and have Ansible installed on the host machine.
 test suite breaks something and fails to revert. Do not use snapshot names
 starting with `ssg_`.
 
+*NOTE*: If you want to use your own domain, make sure `openscap-1.2.15` and
+`qemu-guest-agent` are installed there, `root` is accessible via ssh and that
+packages packages can be installed (for RHEL7, it means subscription enabled).
+For testing Ansible remediations, it is sufficient to have `root` accessible via
+ssh on the libvirt domain and have Ansible installed on the host machine.
+
+### Container backends
+
+There are 2 container backends supported, Podman and Docker.
+
+The container image needs to be prepared so it can accept ssh connections, and scan itself.
+Following services need to be supported:
+
+- `sshd` (`openssh-server` needs to be installed, server host keys have to be in place, root's `.ssh/authorized_keys` are set up with correct permissions)
+- `scp` (`openssh-clients` need to be installed - `scp` requires more than a ssh server on the server-side)
+- `oscap` (`openscap-scanner` - the container has to be able to scan itself)
```

Comment:
Strictly speaking, `scp` and `oscap` isn't a service.

Reply:
Changed to requirements.